### PR TITLE
SearchIndex must know about tags to return them.

### PR DESCRIPTION
**Pull request by cahrens**
I was not aware of the SearchIndex class when I added tags to the model.

### CODE REVIEW

**Comment by cahrens** [comment]
@andy-armstrong and @dan-f Please review. This is a follow-up PR from #22.

(fyi @dianakhuang, who reviewed the first PR)

**Comment by andy-armstrong** [comment]
:+1:

File: notesapi/v1/tests/test_views.py [hunk 1]
```diff
@@ -430,10 +430,16 @@ def test_search(self):
         self.assertEqual(last_note, note)
         self.assertEqual(results['total'], 3)
 
-        results = self._get_search_results(text="Second")
-        self.assertEqual(results['total'], 1)
-        self.assertEqual(len(results['rows']), 1)
-        self.assertEqual(results['rows'][0]['text'], 'Second note')
+        def test_search(searchText, expectedText, expectedTags):
```
Comment (dan-f) [inline notesapi/v1/tests/test_views.py]: I think this should have a different name from the encapsulating method to avoid confusion.

**Comment by dan-f** [comment]
@cahrens - does this PR simply make it so that we return tags in search results, or does it also allow for us to search by tag?

**Comment by cahrens** [comment]
@dan-f It only returns the tags as part of the search result model information. It does not allow searching by tags.

File: notesapi/v1/tests/test_views.py (hunk 1 above)
Comment (cahrens) [inline notesapi/v1/tests/test_views.py]: You may have a point there...

**Comment by dan-f** [comment]
Cool, then :+1: from me, once you rename that function.

**Pull request merged**